docs/setup/development_environment.md

```diff
@@ -95,10 +95,12 @@ This downloads and builds all dependencies for the workspace and displays the he
 
 !!! tip
 
-    You can also install Pepsi into your local system to conviniently use it without rebuilding:
+    You can also install Pepsi into your local system to conveniently use it without rebuilding:
 
     ```
     cargo install --path tools/pepsi
     ```
 
     Pepsi is subsequently installed at `~/.cargo/bin/pepsi`.
+    Don't forget to update it from time to time with `cargo update pepsi` to get the latest features and bugfixes. <br> <br>
```

Comment:
I don't think updating installed binaries works this way...
Cargo update "update[s] dependencies as recorded in the local lock file".

Reply:
I tried it by incrementing the version number and it seemed to work 🤔

Reply:
Well, running that command after changing the version number does fetch some info from the registry and updates the lockfile, but the installed binary won't be updated.

Reply:
Agree, reverted